src/Sentry/buildTransitive/Sentry.targets

```
@@ -9,6 +9,12 @@
     <SentryAttributesFile>Sentry.Attributes$(MSBuildProjectExtension.Replace('proj', ''))</SentryAttributesFile>
   </PropertyGroup>
 
+  <Target Name="_SentryEnsureAndroidEnableAssemblyCompressionDisabled"
+          Condition="$(TargetFramework.Contains('-android')) AND '$(AndroidEnableAssemblyCompression)' == 'false'"
```

```suggestion
          Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android' AND '$(AndroidEnableAssemblyCompression)' == 'false'"
```

`GetTargetPlatformIdentifier` might be a more reliable way to extract the "android" bit from the TFM...